### PR TITLE
Docs: Layout objects

### PR DESCRIPTION
**Pull request by dprokop**
Adds docs for SceneGridLayout and SceneFlexLayout.

### CODE REVIEW

File: docusaurus/website/sidebars.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const sidebars = {
 
   docs: {
     'Getting started': ['getting-started', 'core-concepts'],
+    'Learn Scenes': ['scene-layout'],
```
Comment (torkelo) [inline docusaurus/website/sidebars.js:21]: ah nice, tried to add behavior docs in the behaviors PR but could not figure out how to add a new page and have it show up in the menu

File: docusaurus/docs/scene-layout.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
+---
+id: scene-layout
+title: Building Scene layout
+---
+
+Scenes support two layout types: flex and grid layout. In this guide you will learn hwo tu use and configure `SceneFlexLayout` and `SceneGridLayout`.
```
Comment (torkelo) [inline docusaurus/docs/scene-layout.md:6]:
```suggestion
Scenes support two layout types: flex and grid layout. In this guide you will learn how to use and configure `SceneFlexLayout` and `SceneGridLayout`.
```

File: docusaurus/docs/scene-layout.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+## Flexbox layout
+
+`SceneFlexLayout` allows building flexible scenes with layout driven by browser's CSS flexbox layout.
```
Comment (torkelo) [inline docusaurus/docs/scene-layout.md:10]:
```suggestion
`SceneFlexLayout` allows building flexible scenes with layout driven by browser's CSS flexbox layout. This allows for defining very dynamic layouts where panel widths and heights can adapt to the available space. 
```

**Review by torkelo — APPROVED** [review]

File: docusaurus/docs/scene-layout.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
+---
+id: scene-layout
+title: Building Scene layout
```
Comment (torkelo) [inline docusaurus/docs/scene-layout.md:3]:
```suggestion
title: Building a scene layout
```

The docs use sentence casing :)  

Is there a way to specify shorter menu title that is different from page title?

Comment (torkelo) [reply docusaurus/docs/scene-layout.md:3]: Ah so this is the menu title, the first h1 will be the page title. in that case I would opt for a shorter menu title like "Layouts"

Comment (dprokop) [reply docusaurus/docs/scene-layout.md]: Yeah, changed that

Comment (dprokop) [reply docusaurus/docs/scene-layout.md]: @torkelo you can see here https://github.com/grafana/scenes/pull/152/commits/d7081c86f074fe120e70f590827c41cd632e020d#diff-49e1f75a07ee4a1c5fec378ea9baf47277e85a444e672212b13ab18351da4db5R21 how to achieve this.

Comment (torkelo) [reply docusaurus/docs/scene-layout.md]: @dprokop ah, great good to know!

**Pull request merged**